monitors/end_of_run_monitor.py

```diff
@@ -11,10 +11,12 @@
 from watchdog.observers import Observer
 
 from monitors.settings import (INST_FOLDER, DATA_LOC, SUMMARY_LOC,
-                               LAST_RUN_LOC, LOG_FILE, INSTRUMENTS)
+                               LAST_RUN_LOC, EORM_LOG_FILE, INSTRUMENTS)
 from utils.clients.queue_client import QueueClient
 
-logging.basicConfig(filename=LOG_FILE, level=logging.INFO, format='%(asctime)s %(message)s')
+logging.basicConfig(filename=EORM_LOG_FILE,
+                    level=logging.INFO,
+                    format='%(asctime)s %(levelname)s [%(filename)s:%(lineno)d] %(message)s')
```

Comment:
Note that this format is being standardised in #206 [here](https://github.com/ISISScientificComputing/autoreduce/pull/206/files#diff-811c695e3e4c4625780187cf9cf2c1fbR5)
However as that is not merged yet this is fine to stay - we'll resolve merge conflict once we get round to it

Reply:
Okay, it doesn't really matter. I just thought it would be good to have the level so you can easily search for 'ERROR'.

Reply:
It is a good change. Just a heads that this may change again soon 😄